### DISCUSSION TITLE
Weapon Oneshot cheat

### DISCUSSION THREAD

**Discussion by priestkos** [Glazelf/RiseHax#7] [Ideas]
Once the game releases, a possible option to make a weapon one shot any monster in offline mode for the lulz

**Comment by Glazelf** [comment]
Might be possible, easiest implementation I can think of is by setting one of the values in damage calculation to an absurdly high amount.
Setting monster HP to 0 seems more effective but this is a funny alternative.

Might have to look out with implementing this for ruining the online scene with people abusing this en masse, and people getting banned in the early months of this game's lifespan.

**Reply by priestkos** [reply]
Agreed that using this online could be a huge ban risk and is not morally good. Could implement check for server connectivity and prohibit activation of this particular cheat for devices connected to Nintendo servers.
